R/S3_print.R

```diff
@@ -76,6 +74,20 @@ print_summary_lm_like <- function(x,
   }
   cat("\n")
 
+  if (is.numeric(x[["diagnostic_endogeneity_test"]])) {
+    cat("\nDiagnostic tests:\n")
```

Comment:
Should it be "Diagnostic tests:" or just "Diagnostics:"

Reply:
Diagnostics is probably better, will change.